src/components/CommentList.jsx

```diff
@@ -1,20 +1,22 @@
 import {useEffect, useState} from "react";
 import {useParams} from "react-router-dom";
 import {getCommentsArr} from "../utils/api";
+import CommentAdder from "./CommentAdder";
 import CommentCard from "./CommentCard";
 
 import SingleArticle from "./SingleArticle";
 
 export default function CommentList() {
+  const {article_id} = useParams();
   const [isLoading, setIsLoading] = useState(false);
   const [commentsArr, setCommentsArr] = useState([]);
-  const {article_id} = useParams();
 
   useEffect(() => {
     setIsLoading(true);
     getCommentsArr(article_id).then(({comments}) => {
       const sortedComments = comments.sort((c1, c2) => {
```

Comment:
This should be able to be done using a query on your backend, rather than sorting all the comments after they've come from your API, if you have the functionality. If not, this will do fine :)

Reply:
Yes, I also think the same. But my backend project doesn't have this functionality, because it wasn't in the task. First I wanted to add it to the backend project now, but then I decided to put it here. So can I merge?